### ISSUE TITLE
Unknown Hard Error - Every time after closing the desktop app

### ISSUE THREAD

**Issue by SeriousHoax** [ente-io/ente#4523]
### Description

Every time the Ente Auth desktop app is closed, I'm presented with this error message.
![1](https://github.com/user-attachments/assets/c743495a-57da-43c9-954d-583f71baae20)
The app works fine as expected, so I'm not sure why this error is shown.

### Version

v4.2.1

### What product are you using?

Ente Auth

### What platform are you using?

Desktop - Windows

**Comment by btdat2506** [comment]
[ente-logs-2024-Dec-29-9-10.zip](https://github.com/user-attachments/files/18268970/ente-logs-2024-Dec-29-9-10.zip)

Same problem here. I also upload the log files here.

**Comment by BorJofi** [comment]
Same issue here. I even uninstalled and reinstalled, thinking the problem was with my PC.

**Comment by notgabry** [comment]
any updates?

**Comment by aaskjer** [comment]
have this error too since i'm on Win11 24H2 26100.2605

**Comment by jcrowe37** [comment]
Also reciving this error since updating to version 4.2.4 on Win11 24H2 build 26100.2605

**Comment by zhaojiannet** [comment]
我出现了同样的问题

![WeChat_20250114160410](https://github.com/user-attachments/assets/30d61f80-842d-4950-aabf-26f3b0b26643)


版本	Windows 11 IoT 企业版 LTSC
版本号	24H2
安装日期	‎2025/‎1/‎10
操作系统版本	26100.2605
体验	Windows 功能体验包 1000.26100.36.0

**Comment by god0654** [comment]
Same error here.
Win11 24H2.

**Comment by RJSackville** [comment]
Same problem here. Windows 11 24H2 26100.2894, Ente Auth v4.2.8

**Comment by prateekmedia** [comment]
According to [StackOverflow article](https://stackoverflow.com/questions/13314945/unknown-hard-error-message-is-shown-when-application-exits-using-a-call-to-envir), it is because Windows Error Reporting Service was disabled on the Windows machine. Please let me know if it's still happening even after enabling that service.

**Comment by jcrowe37** [comment]
@prateekmedia Windows Error Reporting Service is definitely disabled on my affected machine, I will enable that service tomorrow when i'm at that machine and report back.

**Comment by SeriousHoax** [comment]
> According to [StackOverflow article](https://stackoverflow.com/questions/13314945/unknown-hard-error-message-is-shown-when-application-exits-using-a-call-to-envir), it is because Windows Error Reporting Service was disabled on the Windows machine. Please let me know if it's still happening even after enabling that service.

You're right, it's related to this. I have my own script that I use after installing Windows which disables it among other things. 
After enabling it, the error goes away. Is there something you can do so that the error is not shown even when the Windows Error Reporting Service is disabled?

**Comment by chaoscreater** [comment]
I don't even see a Windows Error Reporting service on Windows 11 23H2. It was working fine for weeks, if not months. Then I plugged in a new monitor today and then all of a sudden I get this error on closing Ente Auth. I haven't updated my machine or done anything at all, didn't even restart it.

**Comment by ua741** [comment]
This should be hopefully fixed in the latest version v4.4.3
